### DISCUSSION TITLE
Change crescendo step number?

### DISCUSSION THREAD

**Discussion by mutti-wilson** [GrandOrgue/grandorgue#859] [Help]
Is there a way to adjust the number of crescendo steps from 32 to something lower? Often my pedal will be right on the verge of going to the next or previous step and while playing it will shift registration. If I could make it more coarse I believe it would solve the issue.

**Comment by larspalo** [comment]
You can of course try programming the same combination for multiple steps and you'll have that effect.

Otherwise, I'd say that it all boils down to having good MIDI equipment that is well calibrated and it will behave properly.

**Reply by mutti-wilson** [reply]
That's what I am planning on doing. I'll probably boil it down to 10 steps or so.
